app.rb

```diff
@@ -0,0 +1,139 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  attr_accessor :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # list all books
+  def list_all_books
+    if @books.empty?
+      puts 'Sorry, the books list is currently empty'
+    else
+      @books.each do |book|
+        puts "Tilte: \"#{book.title}\", Author: #{book.author}"
+      end
+    end
+  end
+
+  # list all people
+  def list_all_people
+    if @people.empty?
+      puts 'Sorry, the people list is currently empty'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
```

Comment:
Listing people does not display the correct names. In the example below, I created two people. A student `Tom` and a teacher `Sally`. But when I list all people the name displayed for the student is `true` and the name for the teacher is `English`

![image](https://user-images.githubusercontent.com/38088818/197335724-25751b3b-7365-4f41-8200-5c0e3d717434.png)

- Kindly look into it.